mobile/Features/ContentLibrary/Screens/Tag.jsx

```diff
@@ -0,0 +1,236 @@
+import {
+  Text, View, TouchableOpacity, Image, ScrollView,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Swiper from 'react-native-swiper';
+import axios from 'axios';
+import { useContext } from 'react';
+import starImage from '../../../assets/star.png';
+import leftArrow from '../../../assets/left.png';
+import rightArrow from '../../../assets/right.png';
+import shapeImage from '../../../assets/shape.png';
+import cancelImage from '../../../assets/cancel.png';
+import goldStar from '../../../assets/goldStar.png';
+import style from '../Components/ContentStyle';
+import TagContext from '../Context/TagContext';
+
+export default function Tag({ navigation, route }) {
+  /* index of corresponding Tag */
+  const { index, routeName } = route.params;
+
+  const {
+    Tags, deleteFavorite, addFavorite, findFavorite,
+  } = useContext(TagContext);
+
+  /* Grabs current tag */
+  const tag = Tags[index];
+
+  const {
+    _id, tagName, tagBrief,
```

Comment:
I'm getting an error on this line :(
about can't get _id of undefined

i believe I'm also getting a failed prop type warning
that route.params.index is a string and its being passed into "Tag" which expects a number instead.